app/routes/staff/-components/$sidebar.tsx

```diff
@@ -29,7 +29,7 @@ export const sidebarToggleButtonStyles = tv({
   variants: {
     device: {
       mobile: {
-        base: "fixed bottom-5 left-5 rounded-full border bg-bg p-3 transition hover:bg-muted",
+        base: "fixed bottom-5 left-5 z-50 rounded-full border bg-bg p-3 transition hover:bg-muted",
```

Comment:
DrawerやModalはボタンより上に表示したいことを考えると、`z-30`か`z-40`位にしておいたほうが良いと思います

```suggestion
        base: "fixed bottom-5 left-5 z-30 rounded-full border bg-bg p-3 transition hover:bg-muted",
```